content/en/docs/armory-admin/Secrets/secrets-aws-sm.md

```diff
@@ -6,6 +6,40 @@ aliases:
 
 You can configure AWS Secrets Manager as a secrets engine for Spinnaker.  See the AWS Secrets Manager [User Guide](https://docs.aws.amazon.com/secretsmanager/latest/userguide/intro.html) for how to set up AWS Secrets Manager,
 
+## Authorization
+Remember to run the Operator (or Halyard's daemon) and Spinnaker services with IAM roles that allow them to read that content. An example of a policy that can be applied to the role is to allow for access to the SecretsManger and the KMS store
```

Comment:
Remember to run the Operator (or Halyard's daemon) and Spinnaker services with IAM roles that allow those services to read what content?

Reply:
Ah, I was copying from the S3 one.  I'll clarify in both.  Thanks